tools/who_what_benchmark/README.md

```diff
@@ -11,8 +16,46 @@
 * Validation of text-to-image pipelines. Computes similarity score between generated images:
     * Supports Diffusers library and Optimum-Intel via `Text2ImageEvaluator` class.
 
-The main idea is to compare similarity of text generation between baseline and optimized LLMs.
+### Installation
+Install WWB and its requirements from the source using `pip` or any other package manager. For example,
+
+* `python -m venv eval_env`
+* `source eval_env/bin/activate`
+* `pip install -r requirements.txt`
```

Comment:
does requirement.txt installation separately from tool is necessary?

Reply:
I think it is ok for newcomers. Runtime folks came and ask about the tool usage and it was helpful for them as they are not Python users. So I would keep this.